flutter/lib/backend/native.dart

```diff
@@ -1,13 +1,20 @@
 import 'dart:ffi';
 import 'dart:io';
 
+import 'package:flutter/services.dart';
+
 import 'package:device_info/device_info.dart';
 import 'package:ffi/ffi.dart';
 
 import 'package:mlcommons_ios_app/protos/backend_setting.pb.dart' as pb;
 import 'package:mlcommons_ios_app/protos/mlperf_task.pb.dart' as pb;
 import 'bridge.dart';
 
+const androidChannel = MethodChannel('org.mlcommons.mlperfbench/android');
+Future<String> getNativeLibraryPath() async {
+  return await androidChannel.invokeMethod('getNativeLibraryPath') as String;
```

Comment:
will this work for other OS as well, like windows / iOS?

Reply:
This method will throw an exception if we try to run it on Windows/iOS.
We only call it if `Platform.isAndroid` returned true:
https://github.com/mlcommons/mobile_app_open/blob/dea36c31605311a4b98d9d1f43146e479ace4dbd/flutter/lib/benchmark/benchmark.dart#L630-L631

Reply:
I don't know of any equivalent of `nativeLibraryDir` for other platforms.
On iOS we don't use any custom dynamic libraries yet. And we only use tflite backend, and there are no plans on adding more backends, as far as I'm aware.
On Windows the package is simpler that on Android/iOS, and we just keep all `.dll` files near executable.